torch/testing/_internal/common_methods_invocations.py

```diff
@@ -744,8 +744,9 @@ def sample_inputs_binary_pwise(op_info, device, dtype, requires_grad, **kwargs):
     for first_shape, shape_or_scalar, broadcasts_input in test_cases:
         arg = shape_or_scalar
         if isinstance(shape_or_scalar, tuple):
+            exclude_zero = kwargs.get('rhs_exclude_zero', False)
```

Comment:
Let's extend the comment documentation with what kwargs are currently supported and what they do, too